kratos/python/add_processes_to_python.cpp

```diff
@@ -472,6 +473,20 @@ void  AddProcessesToPython(pybind11::module& m)
         .def(py::init<ModelPart&, ModelPart&, Parameters>())
     ;
 
+    py::class_<ApplyRayCastingInterfaceRecognitionProcess<2>, 
+      ApplyRayCastingInterfaceRecognitionProcess<2>::Pointer, Process>
+      (m,"ApplyRayCastingInterfaceRecognitionProcess2D")
+        .def(py::init<ModelPart&, ModelPart&>())
+        .def(py::init<ModelPart&, ModelPart&, Parameters>())
+    ;
+
+    py::class_<ApplyRayCastingInterfaceRecognitionProcess<3>,
+       ApplyRayCastingInterfaceRecognitionProcess<3>::Pointer, Process>
+       (m,"ApplyRayCastingInterfaceRecognitionProcess3D")
+        .def(py::init<ModelPart&, ModelPart&>())
+        .def(py::init<ModelPart&, ModelPart&, Parameters>())
+    ;
+
```

Comment:
Same comment, I'd export the `Model-Parameters` constructor instead.

Reply:
done